main.go

```diff
@@ -301,32 +283,36 @@ func setupEnvironmentVars() error {
 
 	}
 
-	Debug.log("setupEnvironmentVars EXIT")
+	ControllerDebug.log("Appsody Controller envirnonment variables: ", "APPSODY_WATCH_IGNORE_DIR: ", tmpWATCHIGNOREDIR, " APPSODY_RUN_KILL: ", appsodyRUNKILL, " APPSODY_DEBUG_KILL: ", appsodyDEBUGKILL,
```

Comment:
It would be better to store all the variables in a dictionary just print the dict (or loop over the vars to print them)

Reply:
Depends on whether you mean just for printing this out, or the whole main.go file.  I am not sure it is worth doing that.